src/avfoundation.cs

```diff
@@ -12645,49 +13068,64 @@ interface AVContentKeySession_AVContentKeyRecipients
 		IAVContentKeyRecipient[] GetContentKeyRecipients ();
 	}
 
-	[TV (10,2), Mac (10,12,4), iOS (10,3), NoWatch]
+	[TV (10,2), Mac (10,12,4), iOS (10,3), Watch (6,0)]
```

Comment:
that looks like the same situation - that type should not be exposed in watchOS, see
https://developer.apple.com/documentation/avfoundation/avcontentkeyrequest